zowe/zowe-cli
# Normalize capitalization and punctuation in command descriptions and summaries

**What It Does**
Attempts to provide greater consistency in terms of punctuation and capitalization for Zowe CLI command help text. 

General rules applied:
- Capitalize the first letter of all descriptions, summaries, and options.
- End every description with a period.
- End every summary without a period (this requires shortening multi-sentence summaries to a single sentence).

Automating enforcement of these rules may be too complicated unless there is a way to programmatically get all help strings. Scanning the code line-by-line can be easily confounded by use of variables instead of literals.

I left option descriptions and examples mostly as they were.

**How to Test**
Try some zowe commands and look at the help text. Everything should start with a capital letter. Descriptions end with a period. Summaries end without a period unless they are multiple sentences.

**Review Checklist**
I certify that I have:
